src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -545,6 +567,11 @@ const reducer = (state: CheckoutState, action: Action): CheckoutState => {
         confirmationToken: action.payload.confirmationToken,
         steps: newSteps,
       }
+    case "SET_SHOW_ORDER_SUBMITTING_SPINNER":
```

Comment:
We should stick to conventional terminology vs implementation details tied to UI ("spinner" -- what if we use some other kind of loading UI mechanism?).

Something like 
```
ORDER_SUBMITTING
```

Reply:
sweet, that will do it.